example-get-started/code/.github/workflows/cml.yaml

```diff
@@ -14,13 +14,18 @@ jobs:
           echo "# CML Report" > report.md
 
           echo "## Plots" >> report.md
-          dvc plots diff bigrams-experiment workspace \
+          dvc plots diff main \
```

Comment:
Why is the `dvc plots diff` target different than `dvc metrics diff`? It seems like they should either both be comparing against `main` or both be comparing against `HEAD~1`.

(from https://github.com/iterative/dvc/issues/7578#issuecomment-1102545000)